hw/_index.md

```diff
@@ -12,7 +12,7 @@ See the [Hardware Development Stages]({{< relref "/doc/project/hw_stages.md" >}}
 
 Next, we focus on all available [processor cores](#processor-cores) and provide links to their design specifications, DV plans and the DV simulation results.
 
-Finally, we provide the same set of information for all available [top level designs](#top-level-designs). We may eventually also include the synthesis results for those top level designs.
+Finally, we provide the same set of information for all available [top level designs](#top-level-designs), including an additional dashboard with preliminary synthesis results for some of those designs.
```

Comment:
Perhaps `some of these`? Not entirely sure, English is not my first language.

Reply:
I think `these` could indeed be used here since we refer to the designs mentioned in the previous part of this sentence.

Reply:
amended, thanks!